core/src/main/java/fj/data/Enumerator.java

```diff
@@ -185,6 +181,18 @@ private Enumerator(final F<A, Option<A>> successor, final F<A, Option<A>> predec
     return Stream.fromFunction(this, id, a);
   }
 
+  /**
+   * Returns a stream of the values from this enumerator,
+   * starting at the min of given Bounded, ending at the max, counting up.
+   *
+   * @param bounded A value at which to begin the stream.
+   * @return a stream of the values from this enumerator, cut by bounded, counting up.
+   */
+  public Stream<A> toStream(final Bounded<A> bounded) {
+    final F<A, A> id = identity();
```

Comment:
I think the implementation of this method should just delegate to the existing overload:
```
public Stream<A> toStream(final Bounded<A> bounded) {
  return toStream(bounded.min());
}
```